dist/src/main/java/io/camunda/commons/CommonsModuleConfiguration.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.commons;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration(proxyBeanMethods = false)
+@ComponentScan("io.camunda.commons")
+@ConfigurationPropertiesScan("io.camunda.commons")
+@EnableAutoConfiguration
+public class CommonsModuleConfiguration {}
```

Comment:
❓ 💭 Could this be part of the ApplicationModule class?

Reply:
I plan to consolidate these configuration classes to avoid having too many fine-grained, as mentioned above. However, I would tackle it in a separate pull request as mentioned above.

Reply:
I removed the `ApplicationModuleConfiguration` class, and moved the `configuration` into the `commons` package.